microraiden/microraiden/constants.py

```diff
@@ -0,0 +1,64 @@
+"""
+This file contains configuration constants you probably don't need to change
+"""
+import json
+import os
+
+# api path prefix
+API_PATH = "/api/1"
+# absolute path to this directory. Used to find path to the webUI sources
+MICRORAIDEN_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+# webUI sources
+HTML_DIR = os.path.join(MICRORAIDEN_DIR, 'microraiden', 'webui')
+# javascript library for microraiden
+JSLIB_DIR = os.path.join(HTML_DIR, 'js')
+# url prefix for jslib dir
+JSPREFIX_URL = '/js'
+# decimals of the token. Any price that's set for the proxy resources is multiplied by this.
+TKN_DECIMALS = 10**18  # token decimals
+
+# ethereum node RPC interface should be available here
+WEB3_PROVIDER_DEFAULT = "http://127.0.0.1:8545"
+
+# name of the channel manager contract
+CHANNEL_MANAGER_ABI_NAME = 'RaidenMicroTransferChannels'
+# name of the token contract
+TOKEN_ABI_NAME = 'CustomToken'
+# compiled contracts path
+CONTRACTS_ABI_JSON = 'data/contracts.json'
+
+with open(os.path.join(MICRORAIDEN_DIR, 'microraiden', CONTRACTS_ABI_JSON)) as metadata_file:
+    CONTRACT_METADATA = json.load(metadata_file)
+
+# required version of the deployed contract at CHANNEL_MANAGER_ADDRESS.
+# Proxy will refuse to start if the versions do not match.
+MICRORAIDEN_VERSION = "0.1.0"
+CHANNEL_MANAGER_CONTRACT_VERSION = MICRORAIDEN_VERSION
+# they should stay the same until we decide otherwise
+assert MICRORAIDEN_VERSION == CHANNEL_MANAGER_CONTRACT_VERSION
```

Comment:
Does this still makes sense here? If yes, maybe `check_contract_version` can be made more abstract and used here.